functions/ConvertTo-PasswordStateCredential.ps1

```diff
@@ -0,0 +1,50 @@
+﻿<#
+    .SYNOPSIS
+    Convert a password result to a PSCredential.
+
+    .DESCRIPTION
+    Verify the input object as a PasswordResult and convert the object to a PSCredential.
+
+    .EXAMPLE
+    PS C:\> ConvertTo-PasswordStateCredential -PasswordResult (Find-PasswordStatePassword '"testuser"')
+    Returns the test user object including password as a PSCredential object.
+    .EXAMPLE
+    PS C:\> Find-PasswordStatePassword "testuser" | ConvertTo-PasswordStateCredential
+    Returns the test user object including password as a PSCredential object.
+
+    .PARAMETER PasswordResult
+    A PasswordResult object returned by the Find-PasswordStatePassword function
+
+    .OUTPUTS
+    Returns a PSCredential object.
+
+    .NOTES
+    2019 - Jarno Colombeen
+#>
+Function ConvertTo-PasswordStateCredential {
```

Comment:
Looks good, only thing i would change is the verb to ConvertFrom-PasswordstateCredential.
Since you are converting as password state entry to a powershell credential.

Reply:
Or change it to ConvertTo-PSCredential

Reply:
Yeah that works too. Do ConvertTo-PSCredential